src/components/top-bar/top-bar.atoms.tsx

```diff
@@ -0,0 +1,4 @@
+import ReapitLogoIcon from './icons/brand-reapit.svg?react'
```

Comment:
**todo:** As mentioned earlier in this PR, can you please remove the use of `?react` and instead do:

```javascript
import reapitLogoUrl from './icons/brand-reapit.svg'
import type { ImgHTMLAttributes } from 'react'

interface ReapitLogoProps extends Omit<ImgHTMLAttributes<HTMLImageElement>, 'src' | 'srcSet'> {}

export function ReapitLogo(props: ReapitLogoProps) {
  return (
    <img {...props} src={reapitLogoUrl} />
  )
}
```

**note:** The `img` element may need to be a styled component (i.e. `styled.img`) that bakes in some styles to ensure the img fills it's container while maintaining the correct aspect ratio etc, but hopefully the above gives you a better understanding of what we need this `ReapitLogo` atom to do in order to avoid the `?react` query.

Reply:
thanks for the example, I haven't thought to wrap it using `img` previously, its 100% works 👍